util/weekly_build/4_SpackInstall.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -ex
+
+if [ -z $SETUPDONE ]; then . ShellSetup.sh $* ; fi
+
+cd $RUNDIR/${RUNID}
+
+set +x
+. setup.sh
+set -x
+
+INSTALL_OPTS="--show-log-on-error --fail-fast $INSTALL_OPTS"
+
```

Comment:
seeing a problem - infinite chain of links...

Reply:
I'm not sure I follow. INSTALL_OPTS can be set in ShellSetup.sh (for instance I'm using `-j6` on Acorn), so any platform-specific options will get added to the defaults here. Is there a platform where this is not working?

Reply:
if you can login to jenkins .. take a look here:
https://jenkins.epic.oarcloud.noaa.gov/job/spack-stack/job/nightly/7/execution/node/85/ws/orion-intel/weekly/envs/build-intel/

Reply:
It says I don't have overall/read permissions.
It shouldn't be using links at all, so I think I'll need to see the output.

Reply:
That's expected, that's part of the path padding for the build caching. It uses an extra long path in order to guarantee that there's space in, say, a shared library to search-and-replace the new installation path.

Reply:
okay. sure.
So what is supposed to occur during 4_SpackInstall.sh ?
I don't see much happening in the log output.

Reply:
It should be fetching packages (`spack fetch`) then installing (`spack install`). I'm not sure why it wouldn't be showing output, maybe a slow network connection or filesystem? Where does it seem to be stuck?

Reply:
see details in slack.

Reply:
For Orion and Hercules, the problem I see may be related to them both being in the same filesystem, causing collision on pre-cleaning.  Trying a new way to overcome this,

Reply:
Looking at the logs, it appears it's failing with `/usr/bin/tar: Relink '/apps/spack-managed/gcc-11.3.1/intel-oneapi-compilers-2023.1.0-sb753366rvywq75zeg4ml5k5c72xgj72/compiler/2023.1.0/linux/compiler/lib/intel64_lin/libimf.so' with '/lib64/libm.so.6' for IFUNC symbol 'sincosf'`.